client/src/plugins/getConfig.ts

```diff
@@ -65,9 +65,13 @@ export function getConfig (): Promise<EnvConfigI> {
      * authConfig is a workaround to fix the user settings call as it expects a URL with no trailing slash.
      * This will be removed when a fix is made to sbc-common-components to handle this
      */
+    const nameRequestUrl = response.data[0]['NAME_REQUEST_URL']
+    const nroUrl = response.data[0]['NRO_URL']
     const authConfig = {
       'AUTH_URL': businessUrl,
-      'VUE_APP_PAY_ROOT_API': authPayUrl
+      'VUE_APP_PAY_ROOT_API': authPayUrl,
+      'NAME_REQUEST_URL': nameRequestUrl,
+      'NRO_URL': nroUrl
```

Comment:
Why is this needed? Doesn't auth-web read its own config?

Reply:
There is a bug in sbc-auth, discussed with Saravan. See the comment https://github.com/bcgov/entity/issues/5806#issuecomment-742675724